### PR TITLE
Upgrade ruby to 2.4.9

### PR DESCRIPTION
**Pull request by sethkrasnianski**
This addresses some security vulnerabilities.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.0.0'
+ruby '2.4.7'
```
Comment (joealba) [inline Gemfile]: @sethkrasnianski 2.4.9 is available.  Anything preventing us from going up to that patch level?

Comment (sethkrasnianski) [reply Gemfile]: Nope. Lemme update.

Comment (sethkrasnianski) [reply Gemfile]: @joealba updated.

**Review by joealba — APPROVED** [review]
:bingo: :bongo:

**Pull request merged**